my-react-app/src/components/LoginPage.js

```diff
@@ -0,0 +1,39 @@
+import { Auth } from 'aws-amplify';
+import React, { useState } from 'react';
+import './LoginPage.css';
+
+const LoginPage = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+
+  const handleLogin = async () => {
+    try {
+      await Auth.signIn(username, password);
+    } catch (err) {
+      setError(err.message);
+    }
+  };
+
+  return (
+    <div className="login-container">
+      <h1>Login</h1>
+      <input
+        type="text"
+        placeholder="username"
+        value={username}
+        onChange={(e) => setUsername(e.target.value)}
+        />
+        <input
+          type="password"
+          placeholder="password"
+          value={password}
+          onChange={(e) => setPassword(e.target.value)}
+          />
+          <button onClick={handleLogin}>Login</button>
```

Comment:
Consider adding type="button" to the button element to avoid unintended form submissions if your login page is part of a larger form. Otherwise great job. :)